scripts/release/mirror.test.js

```diff
@@ -19,8 +20,10 @@ describe('mirror', () => {
         ['26', '26'],
       ],
       edge: [
-        ['50', '79'],
-        ['78', '79'],
+        ['1', false], // wrong, invalid inference
+        ['27', false], // wrong, invalid inference
+        ['28', '79'], // wrong, could be in Edge <79
+        ['78', '79'], // wrong, could be in Edge <79
```

Comment:
Mirroring shouldn't generate ranges (which is inconsistent with WebView below; I'll be fixing that up).

```suggestion
        ['28', '79'],
        ['78', '79'],
```

Reply:
My point wasn't that we should use ranges, but that for something implemented long before Chrome 79 it might have been in Edge 18 or earlier too.

But in those cases we shouldn't use "mirror" so I no longer think that's a problem.